container_pipeline/cleanup_registry/garbagecollector.py

```diff
@@ -10,53 +12,118 @@ class GarbageCollector(object):
     """
 
     def __init__(self, registry_host="127.0.0.1", registry_port="5000", registry_secure=False, local_index=False,
-                 index_git="https://github.com/centos/container-index", verbose=True, collect=True):
+                 index_git=None, index_location="./c_i", verbose=True, collect=True):
```

Comment:
I think default is already added at https://github.com/CentOS/container-pipeline-service/pull/416/files#diff-37f1187e22ecd347ea22a51bc9ba767bR31 for `index_location`, why to repeat?